phys2bids/tests/test_integration.py

```diff
@@ -0,0 +1,466 @@
+import os
+import wget
+import glob
+import json
+import subprocess
+
+from pkg_resources import resource_filename
+
+from phys2bids.phys2bids import phys2bids
+from phys2bids._version import get_versions
+
+
+def test_logger():
+    """
+    Tests the logger
+    """
+
+    # Move into folder
+    test_path = resource_filename('phys2bids', 'tests/data')
+    subprocess.run(f'cd {test_path}', shell=True, check=True)
+
+    # Input arguments
+    test_filename = 'tutorial_file.txt'
+    test_chtrig = 1
+    test_outdir = test_path
+
+    # Phys2bids call through terminal
+    subprocess.run(f'phys2bids -in {test_filename} -indir {test_path} '
+                   f'-chtrig {test_chtrig} -outdir {test_outdir}', shell=True, check=True)
+
+    # Read logger file
+    files = os.listdir(test_path)
+    logger_idx = [log_idx for log_idx, log_str in enumerate(
+                      files) if 'phys2bids' in log_str]
+    logger_file = files[logger_idx[0]]
+
+    with open(os.path.join(test_path, logger_file)) as logger_info:
+        logger_info = logger_info.readlines()
```

Comment:
Can we reduce this code a bit using e.g., a `glob`?

```suggestion
    # Read logger file
    logger_file = glob.glob(os.path.join(test_path, '*phys2bids*'))[0]

    with open(os.path.join(test_path, logger_file)) as logger_info:
        logger_info = logger_info.readlines()
```